test/lit/passes/outlining.wast

```diff
@@ -524,21 +542,59 @@
     )
   )
 )
-;; CHECK:      (type $0 (func))
 
-;; CHECK:      (type $1 (func (param i32 i32)))
+;; outline if-true.
```

Comment:
It looks like there is already a test for `if` higher up in this file. It would be good to co-locate this test with that one. Also, I don't see any tests for `if-else`. It would be good to add some.